Makefile

```diff
@@ -359,6 +359,18 @@ octant-antrea-ubuntu:
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) projects.registry.vmware.com/antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION)
 
+.PHONY: antrea-mc-controller-distroless
+antrea-mc-controller-distroless:
```

Comment:
I don't think the `distroless` suffix is useful, the image name should just be `antrea-mc-controller`

Reply:
done